_resources/port1.0/group/haskell_cabal-1.0.tcl

```diff
@@ -19,77 +19,124 @@
 # haskell_cabal.bin
 #   The cabal binary. Defaults to ${prefix}/bin/cabal.
 #
-# haskell_cabal.default_args
-#   Default arguments for cabal used across invocations in configure, build, and
-#   destroot phases. Defaults to
-#       --prefix=${prefix}
-#       --bindir=${prefix}/bin
-#       --libdir=${prefix}/lib
-#       --libsubdir=${name}
-#       --dynlibdir=${prefix}/lib
-#       --libexecdir=${prefix}/libexec
-#       --libexecsubdir=${name}
-#       --datadir=${prefix}/share/${name}
-#       --docdir=${prefix}/share/doc/${name}
-#       --htmldir=${prefix}/share/doc/${name}
-#       --sysconfdir=${prefix}/etc/${name}
-#       --enable-documentation
-#       --enable-relocatable
+# haskell_cabal.cabal_root
+#   The root path for cabal. Defaults to ${workpath}/.home/.cabal .
+#
+# haskell_cabal.gcc_location
```

Comment:
comment is incorrect as gcc will almost never be used.

Reply:
Thanks fixed.